accepted/2020/feature-switch.md

```diff
@@ -0,0 +1,113 @@
+# Feature switch
+The functionality available in the .NET Core frameworks is getting bigger and wider with every release. Applications typically don't need or want all of it. With technologies like the linker we have the ability to remove parts of the framework which the app doesn't use. These technologies have limitations base don existing public API patterns and code behavior, sometimes they can't determine what the app really needs. This proposal adds a way to explicitly disable functionality in the framework.
+
+## Goals
+* Ability to declaratively turn-off functionality in the framework (and potentially any library/app)
+* Same behavior all the time - regardless of linker being used or not (`Debug` build behaves the same as published `Release` app).
```

Comment:
"Same behavior all the time"... with regards to the functionality being turned on/off.  I'm clarifying that this isn't addressing the problem of the linker removing functionality that the app may have been relying on via reflection or some such means.

Reply:
I was trying to say that the behavior is consistent regardless of linker being used or not - linker acts as an optimization, not a behavior changing tool. I'll try to rewrite this.